src/lib/battery/battery.h

```diff
@@ -193,7 +193,7 @@ class Battery : public ModuleParams
 		}
 	}
 
-	bool isFloatEqual(float a, float b) const { return fabsf(a - b) > FLT_EPSILON; }
+	bool isFloatEqual(float a, float b) const { return fabsf(a - b) < FLT_EPSILON; }
```

Comment:
We should really do a proper version of this in mathlib with all the special cases handled.

Reply:
You mean https://github.com/PX4/Matrix/blob/master/matrix/helper_functions.hpp#L23-L40 ?

Reply:
Changed to that.